### PR TITLE
Sanitizing for release.

### PR DESCRIPTION
**Pull request by ericnordelo**
This PR does:

- Remove a lot of unnecessary imports
- Move some constants for cleaning modules
- Refactor `unsafe_state` usage to avoid unnecessary repetition of the initialization.
- Refactor some tests to use the constants in test utils.

### CODE REVIEW

File: src/access/accesscontrol/accesscontrol.cairo [hunk 1]
```diff
@@ -138,7 +137,7 @@ mod AccessControl {
     #[generate_trait]
     impl InternalImpl of InternalTrait {
         fn initializer(ref self: ContractState) {
-            let mut unsafe_state = SRC5::unsafe_new_contract_state();
+            let mut unsafe_state = src5_state();
```
Comment (martriay) [inline src/access/accesscontrol/accesscontrol.cairo:140]: why not inlining in the next line like the diff above?

File: src/access/ownable/dual_ownable.cairo [hunk 2]
```diff
@@ -1,21 +1,15 @@
 // SPDX-License-Identifier: MIT
 // OpenZeppelin Contracts for Cairo v0.7.0 (access/ownable/dual_ownable.cairo)
 
-use array::ArrayTrait;
-use array::SpanTrait;
-use core::result::ResultTrait;
```
Comment (martriay) [inline src/access/ownable/dual_ownable.cairo:6]: how did you identify them? language server?

File: src/account/account.cairo [hunk 3]
```diff
@@ -188,7 +169,7 @@ mod Account {
     #[generate_trait]
     impl InternalImpl of InternalTrait {
         fn initializer(ref self: ContractState, _public_key: felt252) {
-            let mut unsafe_state = SRC5::unsafe_new_contract_state();
+            let mut unsafe_state = src5_state();
```
Comment (martriay) [inline src/account/account.cairo:172]: we can inline this one too, right?

File: src/account.cairo [hunk 4]
```diff
@@ -3,9 +3,12 @@ mod dual_account;
...
-use account::TRANSACTION_VERSION;
 use interface::AccountABIDispatcher;
 use interface::AccountABIDispatcherTrait;
 use interface::AccountCamelABIDispatcher;
 use interface::AccountCamelABIDispatcherTrait;
+
+const TRANSACTION_VERSION: felt252 = 1;
+
+// 2**128 + TRANSACTION_VERSION
+const QUERY_VERSION: felt252 = 340282366920938463463374607431768211457;
```
Comment (martriay) [inline src/account.cairo:14]: although i like "cleaning" the `use` area in `account/account.cairo`, i kind of favor having these associated to the actual account impl module as source of truth. for example, in the future i expect us to have different accounts which might have different tx or query versions in the future.

**Comment by martriay** [comment]
Thanks @ericnordelo for this PR, its attention to detail and commitment to excellence :)

File: src/access/accesscontrol/accesscontrol.cairo (hunk 1 above)
Comment (ericnordelo) [reply src/access/accesscontrol/accesscontrol.cairo:140]: to pass `ref`s down, we need to move it into a variable first, in this example that can't be inlined (compilation error).

File: src/access/ownable/dual_ownable.cairo (hunk 2 above)
Comment (ericnordelo) [reply src/access/ownable/dual_ownable.cairo:6]: Manually (after reading a post in the forum saying that most of the corelib Cairo-related improts (not starknet) were not necessary anymore). I'm sure there are more, and at some point the language server should include that feature, but no yet sadly :(

File: src/account.cairo (hunk 4 above)
Comment (ericnordelo) [reply src/account.cairo:14]: Maybe is worth moving them into the module instead of the file? Just a thought, no strong opinion.

Comment (martriay) [reply src/account.cairo:14]: same, no strong opinion between module and file, as long as it's not the module's root

Comment (ericnordelo) [reply src/account.cairo]: Let me know what you think about the update.

File: src/account/account.cairo [hunk 5]
```diff
@@ -27,6 +25,10 @@ mod Account {
     use starknet::get_contract_address;
     use starknet::get_tx_info;
 
+    const TRANSACTION_VERSION: felt252 = 1;
+    // 2**128 + TRANSACTION_VERSION
+    const QUERY_VERSION: felt252 = 0x100000000000000000000000000000001;
```
Comment (martriay) [inline src/account/account.cairo:30]: huh is this the same as `340282366920938463463374607431768211457`? way better representation if so

File: src/tests/account/test_account.cairo [hunk 6]
```diff
@@ -5,8 +5,7 @@ use openzeppelin::account::Account::PublicKeyImpl;
 use openzeppelin::account::Account;
 use openzeppelin::account::AccountABIDispatcher;
 use openzeppelin::account::AccountABIDispatcherTrait;
-use openzeppelin::account::QUERY_VERSION;
-use openzeppelin::account::TRANSACTION_VERSION;
+use openzeppelin::account::Account::{TRANSACTION_VERSION, QUERY_VERSION};
```
Comment (martriay) [inline src/tests/account/test_account.cairo:8]: i like this line particularly, it conveys what i cared about. in the future `account::EthAccount` could have its own and be equally clear by reading this.

**Review by martriay — APPROVED** [review]

File: src/account/account.cairo (hunk 5 above)
Comment (ericnordelo) [reply src/account/account.cairo:30]: It is, yes

**Review by andrew-fleming — APPROVED** [review]
The contracts look nice and cleeeean! LGTM! I left a small comment

**Pull request merged**